config/locales/en.yml

```diff
@@ -30,6 +30,14 @@
 # available at https://guides.rubyonrails.org/i18n.html.
 
 en:
+  settings_menu:
+    profile: Profile
```

Comment:
Currently, we're not being consistent with our usage of quotes in the translation file. While they are strictly speaking not necessary, using them consistently avoids problems with YAML's type coercions. For example, `Yes` will be turned into a boolean `true` and `no` will become `false`.

Reply:
Oh I didn't know YAML does that. TIL 🎉
I'll change these to strings!

Reply:
Yes, it's one of YAML's more... creative features:

![image](https://user-images.githubusercontent.com/47985/136148125-bd8deac6-73e8-40f0-8386-2a2cc48780d2.png)

Reply:
Ok we should make a decision as `i18n-tasks normalize` removes quotes and right now we have an English file and a French file without them https://github.com/forem/forem/blob/cceb175d5d8e09bab9df945f8973a9231e21a4a3/config/locales/en.yml#L8

(we're not enforcing either)